### PR TITLE
Gradient sampling refactor, part2

### PR DESCRIPTION
**Pull request by EmmaRenauld**
Cleaning and preparing tests for gradient sampling.

Main changes:
- Modified option b0_every! Was not very clear!

Work left:
I don't know how to test test_compute_peak_power in scilpy.gradients.test.test_optimize_gradient_sampling.

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-790/2/

File: scilpy/gradients/optimize_gradient_sampling.py [hunk 1]
```diff
@@ -194,19 +216,18 @@ def correct_b0s_philips(points, shell_idx, verbose=1):
...
     Philips scanner (and other) will find the peak power requirements with its
     duty cycle model (this is an approximation) and increase the TR accordingly
-    to the hardware needs. This minimize this effects by:
+    to the hardware needs. This minimizes this effect by:
 
     1) Randomly permuting the non-b0s samples
     2) Finding the peak X, Y, and Z amplitude with a sliding-window
-    3) Compute peak power needed as max(peak_x, peak_y, peak_z)
-    4) Keeps the permutation yielding the lowest peak power
+    3) Computint the peak power needed as max(peak_x, peak_y, peak_z)
```
Comment (karanphil) [inline scilpy/gradients/optimize_gradient_sampling.py:239]: "Computing"

File: scilpy/gradients/optimize_gradient_sampling.py [hunk 2]
```diff
@@ -293,6 +297,7 @@ def compute_min_duty_cycle_bruteforce(points, shell_idx, bvals, ker_size=10,
 
 def compute_peak_power(q_scheme, ker_size=10):
     """
+    toDo: Description of peak power.
```
Comment (karanphil) [inline scilpy/gradients/optimize_gradient_sampling.py:310]: I found this from Guillaume Gilbert : 

Optimize the diffusion gradient table by minimizing the maximum gradient load on any of the 3 axes over a preset temporal window.

This is the function that is minimized in compute_min_duty_cycle_bruteforce, via compute_peak_power.

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-790/4/

**Review by karanphil — APPROVED** [review]
Good job! Much clearer now :)

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-790/7/

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-790/8/

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-790/9/

File: scilpy/utils/bvec_bval_tools.py [hunk 3]
```diff
@@ -6,8 +6,8 @@
 import numpy as np
 
 from scilpy.image.utils import volume_iterator
-from scilpy.gradientsampling.save_gradient_sampling import (save_gradient_sampling_fsl,
-                                                            save_gradient_sampling_mrtrix)
+from scilpy.io.save_gradient_sampling import (save_gradient_sampling_fsl,
```
Comment (arnaudbore) [inline scilpy/utils/bvec_bval_tools.py:9]: name it gradient_sampling instead of save_gradient_sampling

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Review by arnaudbore — APPROVED** [review]
LGTM need to test it quickly.

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**